src/runtime/description-formatter.ts

```diff
@@ -81,7 +81,7 @@ export class DescriptionFormatter {
   // tslint:disable-next-line: no-any
   _combineSelectedDescriptions(selectedDescriptions: ParticleDescription[], options: CombinedDescriptionsOptions = {}) {
     const suggestions = [];
-    selectedDescriptions.map(particle => {
+    selectedDescriptions.forEach(particle => {
```

Comment:
The team's agreement was to use `for (const particle of selectedDescriptions) {` instead of the `forEach` 
I know we haven't been super consistent with this, but I definitely remember this discussion. 
I thought I'd mention it, since there are many being added in this PR.

Reply:
There's hundreds of these in the code, let's save this for a cleanup PR